.travis.yml

```diff
@@ -4,6 +4,6 @@ python:
   - "3.6"
 install:
   - pip install -r requirements-dev.txt
-  - fab bootstrap
+  - CFLAGS='-fPIC' fab bootstrap
```

Comment:
Also, this change is why this branch is breaking on Windows. Microsoft's C compiler toolchain uses a completely different set of flags.

You can look at [this file in my h3-node project](https://github.com/dfellis/h3-node/blob/master/.prebuild.sh) to see how to deal with Microsoft's compiler to accomplish what's needed.

Reply:
The travis build is on xenial and passes because of the presence of the fPIC flag, the appveyor build is the windows one I believe

Reply:
The windows build seems to be failing because cmake cannot find a suitable generator program

Reply:
I think the file @dfellis linked also shows the flags needed to select the right Visual Studio generator on Windows.